src/components/chart/chart.js

```diff
@@ -0,0 +1,65 @@
+import CommonChartOptions from './common-chart-options';
+import SpecificChartOptions from './specific-chart-options';
+import LineChartPlotOptions from './line-chart';
+import Highcharts from 'highcharts';
+
```

Comment:
Looks like we just need to import the accessibility module here to ensure it's being used correctly - tested this locally and it enables the module & screenreader function to read out the chart data as expected:

`import 'highcharts/modules/accessibility';`

Reply:
@janehughes this has been added now - would you mind testing again?